roles/manage_cnp/tasks/add_db.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Validate cnpg_namespace
+  ansible.builtin.fail:
+    fail: msg="The variable cnpg_namespace is not defined and empty"
+  when: (cnpg_namespace is defined) and (cnpg_namespace | length < 1)
+
+- name: Validate pod_name
+  ansible.builtin.fail:
+    fail: msg="The variable 'pod_name' is not defined and empty"
+  when: (pod_name is defined) and (pod_name | length < 1)
```

Comment:
This check could be moved to the `main.yml` instead of including in every file.

Reply:
updated